cds-snc/cppd-medical-report
# Section 1 Translation

# Description

[Trello card:](https://trello.com/c/a46aCfHK/309-people-can-access-prototype-in-en-and-fr) Translates the start section to the application.

# Test Instructions

1. Go to the application
1. If your browser is in EN-xx locale, click `Français` at the top
1. Navigate through the intro section, to get to Section 1
1. See French translation (original EN and FR forms are linked in the Trello card)

# Other Notes

The logic to translate titles is weird, since there isn't a 1:1 translation (i.e. the EN form has more titles than the FR form). We can change that in a later pull request / [Trello task](https://trello.com/c/WqJVeqsh/356-misc-content-tasks). That might be a good first real change to the medical report content as practice PR reviews.
